CRM/Contact/Import/Parser/Contact.php

```diff
@@ -816,12 +817,10 @@ private function formatProfileContactParams(
           }
         }
       }
-      if (in_array($key, ['nick_name', 'job_title', 'middle_name', 'birth_date', 'gender_id', 'current_employer', 'prefix_id', 'suffix_id'])
-        && ($value == '' || !isset($value)) &&
-        ($session->get('authSrc') & (CRM_Core_Permission::AUTH_SRC_CHECKSUM + CRM_Core_Permission::AUTH_SRC_LOGIN)) == 0 ||
-        ($key === 'current_employer' && empty($params['current_employer']))) {
```

Comment:
Current_employer should not show up anymore actually - so could probably go from above too but maybe in master